order/src/main/java/com/yas/order/model/Checkout.java

```diff
@@ -26,14 +28,15 @@
 public class Checkout extends AbstractAuditEntity {
```

Comment:
I think we should add the `checkout item` inside `Checkout` here, we then able to get all the checkout information when get checkout.
Consider add the `List<CheckoutItem>`. Thank you!